app/frontend/generated/graphql.ts

```diff
@@ -705,3 +707,216 @@ export const TeamAcceptInviteDocument = gql`
 export function useTeamAcceptInviteMutation() {
   return Urql.useMutation<TeamAcceptInviteMutation, TeamAcceptInviteMutationVariables>(TeamAcceptInviteDocument)
 }
+
```

Comment:
Frage: Der Code ist in jedem Fall auch auf PROD ?

Reply:
nein, es gibt keine React Komponente die diesen Code importiert und daher kommt es nicht mit ins Bundle.